dolphinscheduler-standalone-server/src/main/resources/application.yaml

```diff
@@ -242,6 +242,9 @@ server:
     mime-types: text/html,text/xml,text/plain,text/css,text/javascript,application/javascript,application/json,application/xml
   jetty:
     max-http-form-post-size: 5000000
+    accesslog:
```

Comment:
Will this configuration logs `@RequestAttribute(value = Constants.SESSION_USER) User loginUser`? Seems there are some credentials in `User` object.

Reply:
> Will this configuration log `@RequestAttribute(value = Constants.SESSION_USER) User loginUser`? Seems there are some credentials in `User` object.

No, it logs the http request parameters, not the controller method's arguments. Obviously credentials are not in the request parameters, they are in headers

Reply:
It also only logs query string, not request body